lms/services/assignment.py

```diff
@@ -196,6 +196,14 @@ def upsert_assignment_groupings(
     def get_by_id(self, id_: int) -> Assignment | None:
         return self._db.query(Assignment).filter_by(id=id_).one_or_none()
 
+    def is_member(self, assignment: Assignment, user: User) -> bool:
```

Comment:
This could eventually include a `role` parameter to check for an specific type of membership.

We are good with any kind for now.